### ISSUE TITLE
PE1

### ISSUE THREAD

**Issue by chongweil907** [nus-cs2030/2021-s1#182]
Just to confirm as long as my code passes 13/29 of the testcases, I am fine for the PE1 modification right ? Thanks

**Comment by domlimm** [comment]
Yup! 13/29 is completion up till level 4

**Comment by marcuspeh** [comment]
Yup 13 test case. Dont really want to say this but this is the third thread opened just to ask this.
